unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/VRMExport/VRMDetailsController.cs

```diff
@@ -0,0 +1,133 @@
+using Cysharp.Threading.Tasks;
+using DCL.Helpers;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Backpack
+{
+    public class VRMDetailsController : IDisposable
+    {
+        private readonly IUserProfileBridge userProfileBridge;
+        private readonly IVRMDetailsComponentView view;
+        private readonly Dictionary<string, NFTDataDTO> nftDataCache = new ();
+
+        public event Action<string, UnequipWearableSource> OnWearableUnequipped;
+        public event Action<string, EquipWearableSource> OnWearableEquipped;
+
+        public VRMDetailsController(
```

Comment:
We are missing unit tests for this and the rest of the added classes

Reply:
I ran out of time to close this work in the cycle, I take you suggestion and note it as a tech debt for a later PR 👌